src/lbaf/Execution/lbsTransferStrategyBase.py

```diff
@@ -65,8 +65,8 @@ def _get_ranks_to_traverse(self, ranks: list, known_peers: list):
         # Iterate over all provided ranks
         for r_src in ranks:
             # Ignore ranks without migratable objects
-            if not r_src.get_migratable_objects():
-                continue
+            #if not r_src.get_migratable_objects():
+            #    continue
```

Comment:
Can these lines be deleted?

Reply:
Oh no -- please remove the comment `#`s
This is also a leftover debugging hack. But indeed if no objects are migratable the rank should be skipped